### DISCUSSION TITLE
In the prior versions you could easily just add folders that contained models you got from hugging face directly

### DISCUSSION THREAD

**Discussion by pm4user444** [oobabooga/text-generation-webui#856] [Q&A] [answered]
In this version, I don't think it works like that.  I tried copying a folder from a saved location to the model folder and it's not pickable. Am I missing something?  Thanks

**Comment by mcmonkey4eva** [comment]
That should still work the same.

What may have happened is...

(A) you just downloaded the model with the webui still running, in which case you need to click the Refresh button next to the model list
(B) you downloaded a model folder to the wrong location (anything that is not `models/`)
(C) you downloaded a model folder that contains subfolders and you need to sort out which is the proper folder to use
(D) you downloaded a model that has multiple versions within the folder. When this happens the model can't be loaded as the webui doesn't know which version to use. If it's a 4bit model, make sure there is only exactly one `.pt` or `.safetensors` file. If there's multiple, you need to figure out which one is correct and remove the others. (Usually the correct one is labeled something like `4bit-128g-cuda` or similar).

**Reply by pm4user444** [reply, marked as answer]
Thanks, the problem seems to be that the older version worked fine with .bin and the new version wants .pt files when I have the bat file appended with --wbits 4 --groupsize 128.  When I remove those, it works with .bin again but really slowly for some reason.
